kmip/pie/objects.py

```diff
@@ -599,3 +599,158 @@ def __ne__(self, other):
             return not (self == other)
         else:
             return NotImplemented
+
+
+class Certificate(CryptographicObject):
```

Comment:
Is there a way to mark this abstract or anything? We should add a note that this should not be invoked directly but use one of the subclasses.